python/paddle/device/cuda/__init__.py

```diff
@@ -19,6 +24,17 @@
 
 from .streams import Event, Stream
 
+if TYPE_CHECKING:
+    from paddle import CUDAPlace
+    from paddle.base.libpaddle._gpuDeviceProperties import (
+        _gpuDeviceProperties,  # type: ignore
+    )
+
+    _CudaPlaceLike: TypeAlias = Union[
+        "CUDAPlace",
```

Comment:
```suggestion
        CUDAPlace,
```

都在 `if TYPE_CHECKING:` 内，这里可以不需要引号 ～

Reply:
收到